include/SFML/Graphics/Texture.hpp

```diff
@@ -110,15 +110,6 @@ class SFML_GRAPHICS_API Texture : GlResource
     ////////////////////////////////////////////////////////////
     /// \brief Load the texture from a file on disk
     ///
-    /// This function is a shortcut for the following code:
-    /// \code
-    /// sf::Image image;
-    /// if (!image.loadFromFile(filename))
-    ///     return false;
-    /// if (!texture.loadFromImage(image, area))
-    ///     return false;
-    /// \endcode
```

Comment:
It felt odd that our docs basically spell out the implementation. The issue with this is what when the implementation changes (as is the case in this PR) these docs need updating. I thought it better to remove this than continue to keep it up to date as the implementation evolves.

Reply:
+1